### PR TITLE
feat(curriculum): add English curriculum block 8.2 updates and plans on tasks or projects

### PR DESCRIPTION
**Pull request by miyaliu666**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-updates-and-plans-for-tasks-and-projects/6613d00727a7a64a5e010243.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: Brian, it's time to _ our plans for the _ project. What do you _ we should do?
+-->
+
+# --description--
+
+Practice time!
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-updates-and-plans-for-tasks-and-projects/6613d00727a7a64a5e010243.md:15]: Do we not want to use our standard "Fill in the blanks in the dialogue below"?

File: curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-updates-and-plans-for-tasks-and-projects/6613d0773359964ab8812659.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+---
+
+<!--
+AUDIO REFERENCE:
+Sarah: Brian, it's time to discuss our plans for the upcoming project. What do you think we should do?
+-->
+
+# --description--
+
+Practice time.
```
Comment (naomi-lgbt) [inline curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-talk-about-updates-and-plans-for-tasks-and-projects/6613d0773359964ab8812659.md:15]: And for these, "Answer the question below"?

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]

**Review by moT01 — COMMENTED** [review]
I hope you don't mind @miyaliu666, I pushed some minor fixes. Take a look at the diff to see what I changed, or let me know if you want me to show you.

This looks good. The only concern I had is that the first two task descriptions are the "Practice time" style. Do we want to start with two tasks that don't explain anything and just throw users into it? I'm fine with it, just mentioning it.

**Comment by miyaliu666** [comment]
@naomi-lgbt I used "Fill in the blanks in the dialogue below" and "Please answer the question below" in Part 9 and 5 after I saw your comments here.

I'll add explanations for words in the first two tasks as @moT01 pointed out. Those words have been taught in previous blocks, but we can review them here because people may not learn the blocks in strict order.

**Review by moT01 — APPROVED** [review]
LGTM 🎉

**Review by naomi-lgbt — APPROVED** [review]

**Pull request merged**